### DISCUSSION TITLE
Start Menu - Show tiles only

### DISCUSSION THREAD

**Discussion by addison74** [valinet/ExplorerPatcher#2221] [Q&A]
First of all, congratulations for this great piece of software that is able to put in place what Microsoft inexplicably left out. Unfortunately, the update package announced for the next period does not offer these features, which shows a stubbornness in making the transition from Windows 10 to 11 as easy as possible. Most fans of version 10 expect a miracle coming from version 12.

**Regarding the Windows 10 Start Menu.**

In **Settings > Personalization > Start** there is the option **Show app list in Start menu**. Do we have the possibility to display either the list of applications or the tiles, not together as they are now?

![applist](https://github.com/valinet/ExplorerPatcher/assets/8360474/35f7dcbb-d5c8-44ba-baa6-b435f9a96c0b)

Windows 10 offers two buttons at the top to make this switch. Can they be displayed?

![pinned](https://github.com/valinet/ExplorerPatcher/assets/8360474/17b623ad-a984-44d0-b148-b5163da33735)

In **Settings > Personalization > Start** there is the option **Use Start full screen**. Can we use this feature in Windows 11?

Thank you.

**Comment by Amrsatrio** [comment]
Yes, yes, and yes. Please explore the options that EP offers.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/0fe4b0b2-b36a-43c1-9b65-2f8299445587)
